community/mumble/APKBUILD

```
@@ -2,66 +2,51 @@
 # Contributor: Johannes Matheis <jomat+alpinebuild@jmt.gr>
 # Maintainer: Johannes Matheis <jomat+alpinebuild@jmt.gr>
 pkgname=mumble
-pkgver=1.2.19
-pkgrel=9
+pkgver=1.3.0_rc1
+_pkgver=${pkgver/_/-}
+__pkgver=${pkgver/_rc1/}
+pkgrel=0
 pkgdesc="Low-latency, high quality voice chat software"
 url="https://wiki.mumble.info"
 arch="all"
 license="BSD-3-Clause"
 options="!check"
-depends="qt-sqlite"
-depends_dev=""
-makedepends="qt-dev libsndfile-dev boost-dev avahi-dev protobuf-dev
-	libxi-dev speex-dev speexdsp-dev libcap-dev"
+depends="qt5-qtbase-sqlite"
+makedepends="qt5-qtbase-dev libsndfile-dev boost-dev avahi-dev protobuf-dev
+	libxi-dev speex-dev speexdsp-dev libcap-dev alsa-lib-dev qt5-qtsvg-dev
+	qt5-qttools-dev opus-dev
+	"
 install="murmur.pre-install"
 pkgusers="murmur"
 pkggroups="murmur"
 subpackages="$pkgname-doc murmur murmur-doc:murmur_doc:noarch"
-source="https://github.com/mumble-voip/$pkgname/releases/download/$pkgver/$pkgname-$pkgver.tar.gz
-	CVE-2018-20743.1.patch
-	CVE-2018-20743.2.patch
-	boost-1.66.patch
+source="https://github.com/mumble-voip/$pkgname/releases/download/$_pkgver/$pkgname-$_pkgver.tar.gz
 	murmur.initd"
-builddir="$srcdir"/$pkgname-$pkgver
-
-# secfixes:
-#   1.2.19-r9:
-#     - CVE-2018-20743
+builddir="$srcdir/$pkgname-${pkgver/_rc1/}"
```

Shouldn't this be `$__pkgver`?
```suggestion
builddir="$srcdir/$pkgname-$__pkgver"
```

---

No. The `_rc1` tarball contains the directory without the `_rc1` suffix. Not sure why, but that is how it is